test-karate/src/test/java/graphql/ftm/publishPage.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Add pieces of content then Publish the Page
+  Background:
+
+    * def validateNoErrors =
+      """
+      function (response) {
+        const errors = response.errors;
+        if (errors) {
+          return errors;
+        }
+        return [];
+      }
+      """
+
+   * def page_id = __arg.page_id
+   * def content1_id = __arg.content1_id
+   * def content2_id = __arg.content2_id
+   * def container_id = __arg.container_id
+
+  Scenario: Create a new version of a piece of content
+    Given url baseUrl + '/api/v1/page/'+page_id+'/content'
+    And headers commonHeaders
+    And request
+      """
+      [
+        {
+          "contentletsId": ["#(content1_id)", "#(content2_id)"],
+          "identifier": "#(container_id)",
+          "uuid": "1"
+        }
+      ]
+      """
+    When method POST
+    Then status 200
+    * def errors = call validateNoErrors response
```

Comment:
**Just an idea:** Since we are just starting with these tests, we might need these two lines for error validation in multiple tests. Perhaps we can create a new "utils" class and import it via "karate-config" to reuse it across multiple tests instead of declaring it in each test

Reply:
That is a good observation pretty much every single feature within this test can be reused not only functions. 
We need to revisit and reorganize a little bit.  And move stuff to a commons package. But for now, I'm relocating it into this test helpers.feature

Reply:
we need to find the proper layout to share common features and code agreed.